src/main/routes/routes.ts

```diff
@@ -23,6 +23,9 @@ export default function(app: Application): void {
   app.get('/otp-login', app.locals.container.cradle.otpLoginController.get);
   app.get('/otp-login-testing', cors(corsOptions), app.locals.container.cradle.otpLoginTestingController.get);
   app.post('/otp-login', app.locals.container.cradle.otpLoginController.post);
+  app.get('/idam-signin', cors(corsOptions), app.locals.container.cradle.idamSigninController.get);
```

Comment:
We don't need the cors options on this one (the corsOptions is specific to azure)